### PR TITLE
add maximum-paths to bgp_af

### PR DESCRIPTION
**Pull request by andy91315**
1. pass the minitest default vrf and vrf ipv4 and ipv6 unicast
2. pass static analysis

### CODE REVIEW

File: tests/test_bgp_af.rb [hunk 1]
```diff
@@ -610,6 +610,102 @@ def test_dampening
...
+    asn = '55'
+    vrf = 'default'
+    af = 'ipv6 unicast'
+    maximum_paths(asn, vrf, af)
+
+    asn = '55'
+    vrf = 'red'
+    af = 'ipv6 unicast'
+    maximum_paths(asn, vrf, af)
+  end
```
Comment (chrisvanheuveln) [inline tests/test_bgp_af.rb]: I know you got this code pattern from other places in this file but in this case I'd prefer that you simplify it. You can replace everything in this method with something like this:

```
vrfs = ['default', 'red']
afs = [%w(ipv4 unicast), %w(ipv6 unicast)]
vrfs.each do |vrf|
  afs.each do |af|
    maximum_paths(55, vrf, af)
  end
end
```

File: tests/test_bgp_af.rb [hunk 2]
```diff
@@ -610,6 +610,102 @@ def test_dampening
...
+    assert_equal(bgp_af.maximum_paths, val,
+                 'Error: maximum paths value not match to set value')
+
+    val = bgp_af.default_maximum_paths
+    bgp_af.maximum_paths = val
+    assert_equal(bgp_af.maximum_paths, val,
+                 'Error: maximum paths value not match to default value')
+  end
+
+  def test_maximum_paths
```
Comment (chrisvanheuveln) [inline tests/test_bgp_af.rb:637]: I know you got this code pattern from other places in this file but in this case I'd prefer that you simplify it. You can replace everything in this method with a nested loop like this:

```
vrfs = ['default', 'red']
afs = [%w(ipv4 unicast), %w(ipv6 unicast)]
vrfs.each do |vrf|
  afs.each do |af|
    maximum_paths(55, vrf, af)
  end
end
```

That also makes it easier to add additional test cases for other AFs.

These comments apply to the ibgp tests too.

File: tests/test_bgp_af.rb [hunk 3]
```diff
@@ -610,6 +610,102 @@ def test_dampening
   # rubocop:enable Metrics/AbcSize,Metrics/MethodLength
 
   ##
+  ## maximum_paths
+  ##
+  def maximum_paths(asn, vrf, af)
+    /ipv4/.match(af) ? af = %w(ipv4 unicast) : af = %w(ipv6 unicast)
```
Comment (chrisvanheuveln) [inline tests/test_bgp_af.rb]: See the comment I left on line 638. If you implement that then you can delete this line on 616.

**Comment by andy91315** [comment]
change the test code

**Pull request merged**